src/main/scala/rps/Game.scala

```diff
@@ -1,33 +1,55 @@
 package rps
 import scala.io.StdIn.readLine
 
+sealed abstract class Move {
```

Comment:
No particular reason, but I'd rather use a `sealed trait` instead of a `sealed abstract class`. It's definitely more common.